features/guidance/turn-lanes.feature

```diff
@@ -936,48 +936,52 @@ Feature: Turn Lane Guidance
     Scenario: Partitioned turn, Slight Curve - maxspeed
         Given the node map
             """
-                f   e
-                |   |
-                |   |
-                |   c
-            a - b ' |
-                g   d
+                    f   e
```

Comment:
The mini map was updated so there would be an intersection prior to the turn so `end of road` would be marked. I think  before the `end of road` was being marked on a false intersection - since the turn is at the intersection